.gitattributes

```diff
@@ -1,6 +1,3 @@
-web/vtadmin/src/proto/** linguist-generated=true
-web/vtadmin/src/proto/*.js -diff
-web/vtadmin/src/proto/*.ts -diff
```

Comment:
@notfelineit Should we revert this now that the issue was found?

Reply:
Yes! 😄